docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tips for Scaling Rancher
+---
+
+This guide aims at introducing challenges and approaches that should be considered to scale Rancher setups. As setups grow performance will reduce. There are steps we can take to minimize the load that is put on Rancher and optimize Rancher's ability to handle increased loads.
+
+## General Tips on Optimizing Rancher's Performance
+* It is advisable to keep Rancher up to date with patch releases. Performance improvements and bug fixes are made throughout the life of a minor release. You can review the release notes to help inform your own decisions on whether an upgrade is necessary but we recommend keeping yourself up to date in most cases.
+
+* The geographically closer Rancher's infrastructure and downstream cluster's infrastructure are the better. If a user or organization requires clusters/nodes all over the world or spread across many regions, it is best to use multiple Rancher installations.
+
+## Minimizing Load on the local cluster
+The largest bottleneck when scaling Rancher is kubernetes resource growth in the local cluster. The local cluster contains information for all downstream clusters. Many operations that apply to downstream clusters will create new objects in the local cluster and require computation from handlers running in the local cluster.
```

Comment:
```suggestion
The largest bottleneck when scaling Rancher is resource growth in the local Kubernetes cluster. The local cluster contains information for all downstream clusters. Many operations that apply to downstream clusters will create new objects in the local cluster and require computation from handlers running in the local cluster.
```

Reply:
Should we make a reference to existing scaling/performance caveats of upstream K8s? You can see this in the files at https://github.com/kubernetes/community/tree/master/sig-scalability/configs-and-limits

Reply:
I think that'd be good. We often defer to them. Though I would put it up above in the general section.